### PR TITLE
feat: Added asking whether to rewrite files

### LINKED ISSUE #116: Prevent docsify init ./docs from destroying existing initialization in that folder
Running 
```
docsify init ./docs
```
will destroy existing documentations

### PR DESCRIPTION
**Pull request by sy-records**
close #116

### CODE REVIEW

**Review by jthegedus — APPROVED** [review]
LGTM

**Review by anikethsaha — CHANGES_REQUESTED** [review]
LGTM

optional : can we add a test for this ?

**Review by jthegedus — APPROVED** [review]

File: lib/commands/init.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 const fs = require('fs')
 const cp = require('cp-file').sync
 const chalk = require('chalk')
+const inquirer = require('inquirer')
```
Comment (jamesgeorge007) [inline lib/commands/init.js:6]: I think it is better to use [enquirer](https://github.com/enquirer/enquirer) which is a lightweight alternative with a similar API.

Comment (sy-records) [reply lib/commands/init.js:6]: Let me see

Comment (sy-records) [reply lib/commands/init.js]: Done, But there was one problem.

Mac OS and CentOS symbols are inconsistent, causing CI to not pass. https://travis-ci.org/github/docsifyjs/docsify-cli/jobs/719497023

image
image

@jamesgeorge007 @anikethsaha enquirer doesn't support defining symbols yet, what should we do?

image

Comment (sy-records) [reply lib/commands/init.js]: Done.

**Review by anikethsaha — APPROVED** [review]

**Pull request merged**